delivery_carrier_package_measure_required/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _check_required_package_measurement(self, package=None):
+        """Check the required measurement on destination pacakges are set.
+
+        If package is None all result packages on the stock picking are check
```

Comment:
hmm all packages are validated anyway.... outdated docstring or wrong code?

Reply:
There is a typo in docstring `check` -> `checked` but the code does what is said IMO.
From the doc on docstring operation

> set1 & set2 returns the intersection of two recordsets, a new recordset containing only records present in both sources

Reply:
Ok. so basically you are validating that a package belongs to the same picking and, if not, it will fail silently.

Then, I wonder if we really need `package=` to be there since if you want to validate a package you can simply call `package._check_required_dimension()`.
Thus, I'd say that if we keep it on the picking, the package must be validated. Eg:
```
if package and not packages:
    raise exceptions.UserError(_("Picking %s does not contain the package %"))
```
Do you see my point?